ash_shell.Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:latest
```

Comment:
@Arpan-206 this is a lot of stuff added to the root of the repository - two dockerfiles, two scripts. Can we try to place these in a more appropriate place? Something like `local_testing/`?